### ISSUE TITLE
Both Scroll and Fill Layout for Multiple Pages

### ISSUE THREAD

**Issue by sheep96** [rstudio/flexdashboard#181]
The vertical layout for a dashboard may be set in the YAML header with either "vertical_layout: fill" or "vertical_layout: scroll." For multi-page dashboards, however, is it possible to set some pages to "fill" and others to "scroll?" I have been unable to find an option or attribute to achieve this effect.

**Comment by sheep96** [comment]
After playing with CSS styling a bit, I believe I have arrived at a passable solution. Here I use a combination of CSS styling in the Page header and in the Column header. This ensures that the particular page scrolls even though remaining pages all fill the screen.

~~~~
Page 3 {style="position:relative;"}
========================================================

Column {style="height:100pc;"}
--------------------------------------------------------

``` {r}
plot(iris[1:4])
```
~~~~

**Comment by ldfreight** [comment]
I am also interested in such feature.
The best would be to have something similar to orientation, where we can specify it

> Page 2 {data-orientation=rows}
> =====================================

**Comment by oliveraj20** [comment]
Yes - I have tried a few workarounds and nothing seems to work. A feature like what @ldfreight suggested would be great.

**Comment by GH-jfee** [comment]
Adding on that it would be great to have this feature!
